pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>no.digipost.signature</groupId>
 	<artifactId>signature-api-specification</artifactId>
-	<version>1.0-SNAPSHOT</version>
+	<version>1.0-STATUS-API-SNAPSHOT</version>
```

Comment:
We might want to automatize this if it becomes a frequent pattern. Branching across 3 or even 4 repositories, setting and building new versions sounds cumbersome to do manually, but should be easy with a subscript.